### ISSUE TITLE
GEBCO default (absent) _reg suffix results in error 

### ISSUE THREAD

**Issue by liamh** [GenericMappingTools/gmt#6915]
**Description of the problem**
An error occurs when using the [GEBCO relief maps](https://www.generic-mapping-tools.org/remote-datasets/earth-gebco.html) from the server with no __reg_ suffix, which is stated as being optional.

**Full script that generated the error**
``` sh
gmt begin test; gmt grdimage @earth_gebco_20m -JG-0.985605/-9.000000/10; gmt end
```

**Full error message**

```
grdimage [NOTICE]: Remote data courtesy of GMT data server OCEANIA [https://oceania.generic-mapping-tools.org]

grdimage [NOTICE]: GEBCO Earth Relief at 20x20 arc minutes reduced by Gaussian Cartesian filtering (37.1 km fullwidth) [GEBCO Compilation Group, 2021 GEBCO 2021 Grid].
grdimage [NOTICE]:   -> Download grid file [755K]: earth_gebco_20m_p.grd
grdimage [ERROR]: Libcurl Error: HTTP response code said error
grdimage [WARNING]: You can turn remote file download off by setting GMT_AUTO_DOWNLOAD off.
grdimage [ERROR]: File /home/healy/.gmt/cache/earth_gebco_20m.grd was not found
grdimage [ERROR]: Cannot find file /home/healy/.gmt/cache/earth_gebco_20m.grd
grdimage [ERROR]: File /home/healy/.gmt/cache/earth_gebco_20m.grd was not found
grdimage [ERROR]: Cannot find file /home/healy/.gmt/cache/earth_gebco_20m.grd
grdimage [ERROR]: File /home/healy/.gmt/cache/earth_gebco_20m.grd not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
```

**Actual outcome**

No file is produced.

**Expected outcome**

When a suffix `_g` is added,
```
gmt begin test; gmt grdimage @earth_gebco_20m_g -JG-0.985605/-9.000000/10; gmt end
grdimage [NOTICE]: Remote data courtesy of GMT data server OCEANIA [https://oceania.generic-mapping-tools.org]

grdimage [NOTICE]: GEBCO Earth Relief at 20x20 arc minutes reduced by Gaussian Cartesian filtering (37.1 km fullwidth) [GEBCO Compilation Group, 2021 GEBCO 2021 Grid].
grdimage [NOTICE]:   -> Download grid file [757K]: earth_gebco_20m_g.grd
```
an image [test.pdf](https://github.com/GenericMappingTools/gmt/files/9183129/test.pdf) is produced. This outcome is expected without the suffix as well.

**System information**

* Operating system:
   ```
           Chassis: vm
    Virtualization: vmware
  Operating System: Debian GNU/Linux 11 (bullseye)
            Kernel: Linux 5.10.0-15-amd64
      Architecture: x86-64
   ```
* GMT version (`gmt --version`):
6.1.1

**Comment by Esteban82** [comment]
It works fine here. I think that you are trying to use a feature that was released in a newer version (than 6.1.1).  Try to use 6.4.


![test](https://user-images.githubusercontent.com/17908951/180831635-88f963ea-37ae-4883-9044-2f441521c1e0.png)

**Comment by PaulWessel** [comment]
On the[ GMT documentation page ](https://docs.generic-mapping-tools.org/latest/)you can select (from the top left sidebar) which version you want documentation for.   The default is of course the current but you can find 6.1 there and it will not have the feature you were trying.
